components/definitions/newTab.d.ts

```diff
@@ -184,6 +184,12 @@ declare namespace NewTab {
     depositInfoSaved: boolean
   }
 
+  export interface GeminiAssetAddress {
```

Comment:
nit: this type doesn't need to be global (same for most types in this file which is why just a nit). It can be in `gemini_actions.ts`

Reply:
Addressed by https://github.com/brave/brave-core/pull/10998/commits/45722192b52e1b1020e7423cd1dcda4fd43aa7a6